src/App.module.css

```diff
@@ -174,11 +183,8 @@
 .sidePanels {
   gap: 10px;
   flex-grow: 1;
-
-  /* TODO: This is copied from CanvasWrapper and should be shared sizing behavior. */
-  max-width: calc(min(100vw - 75px, 730px));
-  width: fit-content;
   min-width: 300px;
+  max-width: var(--content-width);
```

Comment:
naive css question: how does .sidePanels see the var that is scoped to .contentPanels?

Reply:
Where it's used in the JSX, the `.sidePanels` div is a child of the `.contentPanels` div. CSS variables flow down from parents to children!